envVariables.js

```diff
@@ -132,6 +132,11 @@ let enviromentVariables = {
 		optional: false,
 		default: 'USER',
 	},
+	DEFAULT_CATEGORY_EXTERNAL_ID: {
```

Comment:
@Prajwal17Tunerlabs can we call it : DEFAULT_PROJECT_CATEGORY , anyway we are mentioning it is external id in sample file